tasks/section5.yml

```diff
@@ -151,15 +190,77 @@
       - rule_5.1.8
 
 - name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
-  command: /bin/true
+  file:
+      dest: /etc/at.deny
+      state: absent
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
+  file:
+      dest: /etc/at.allow
+      state: touch
+      owner: root
+      group: root
+      mode: 0600
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | AUDIT | Ensure at/cron is restricted to authorized users"
+  shell: stat -L /etc/cron.deny > /dev/null
+  register: result
+  always_run: yes
+  changed_when: no
+  ignore_errors: yes
+  tags:
+      - level1
+      - level2
+      - audit
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | AUDIT | Ensure at/cron is restricted to authorized users"
+  shell: stat -L -c "%a %u %g" /etc/cron.allow | egrep ".00 0 0"
+  register: result
+  always_run: yes
+  changed_when: no
+  ignore_errors: yes
+  tags:
+      - level1
+      - level2
+      - audit
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
+  file:
+      dest: /etc/cron.deny
+      state: absent
+  tags:
+      - level1
+      - level2
+      - patch
+      - rule_5.1.8
+
+- name: "SCORED | 5.1.8 | PATCH | Ensure at/cron is restricted to authorized users"
+  file:
+      dest: /etc/cron.allow
+      state: touch
+      owner: root
+      group: root
+      mode: 0600
   tags:
       - level1
       - level2
       - patch
       - rule_5.1.8
 
 - name: "SCORED | 5.2.1 | AUDIT | Ensure permissions on /etc/ssh/sshd_config are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/ssh/sshd_config | egrep ".00 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*